var/spack/repos/builtin/packages/rpp/package.py

```diff
@@ -45,9 +45,11 @@ def url_for_version(self, version):
     variant("opencl", default=False, description="Use OPENCL as the backend")
     variant("hip", default=True, description="Use HIP as backend")
     variant("cpu", default=False, description="Use CPU as backend")
+    variant("add_tests", default=False, description="add utilities folder")
```

Comment:
The name of the variant doesn't match with the description

Reply:
I changed the description to "add utilities folder which contains rpp unit tests". I named the variant add_tests instead of add_utilities since there's also an add_tests variant in the mivisionx package. But I could change the variant name to add_utilities here if you think that's more appropriate.